src/main/target/link/stm32_flash_f746.ld

```diff
@@ -0,0 +1,28 @@
+/*
+*****************************************************************************
+**
+**  File        : stm32_flash_f746.ld
+**
+**  Abstract    : Linker script for STM32F746VGTx Device with
+**                1024KByte FLASH, 320KByte RAM
+**
+*****************************************************************************
+*/
+
+/* Entry Point */
+ENTRY(Reset_Handler)
+
+/* Specify the memory areas */
+MEMORY
+{
+    FLASH (rx)        : ORIGIN = 0x08000000, LENGTH = 768K
+    FLASH_CONFIG (r)  : ORIGIN = 0x080C0000, LENGTH = 256K
+
+    TCM (rwx)         : ORIGIN = 0x20000000, LENGTH = 64K
```

Comment:
per page 20, looks like this is tcm: http://www.st.com/content/ccc/resource/technical/document/application_note/0e/53/06/68/ef/2f/4a/cd/DM00169764.pdf/files/DM00169764.pdf/jcr:content/translations/en.DM00169764.pdf